bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZATo2LA01j_5f_pta130_NLO_FXFX/ZATo2LA01j_5f_pta130_NLO_FXFX_run_card.dat

```diff
@@ -0,0 +1,154 @@
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1500 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+    20 = nevt_job! Max number of events per job in event generation. 
+                 !  (-1= no split).
+#***********************************************************************
+# Normalize the weights of LHE events such that they sum or average to *
+# the total cross section                                              *
+#***********************************************************************
+ average = event_norm      ! average or sum
+#***********************************************************************
+# Number of points per itegration channel (ignored for aMC@NLO runs)   *
+#***********************************************************************
+ 0.01   = req_acc_FO       ! Required accuracy (-1=ignored, and use the 
+ 	                   ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+ 5000   = npoints_FO_grid  ! number of points to setup grids
+ 4      = niters_FO_grid   ! number of iter. to setup grids
+ 10000  = npoints_FO       ! number of points to compute Xsec
+ 6      = niters_FO        ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+     0    = iseed       ! rnd seed (0=assigned automatically=default))
+#***********************************************************************
+# Collider type and energy                                             *
+#***********************************************************************
+    1   = lpp1    ! beam 1 type (0 = no PDF)
+    1   = lpp2    ! beam 2 type (0 = no PDF)
+ 6500   = ebeam1  ! beam 1 energy in GeV
+ 6500   = ebeam2  ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+  lhapdf   = pdlabel   ! PDF set
+ 292200    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
please use the recommended settings
https://twiki.cern.ch/twiki/bin/view/CMS/QuickGuideMadGraph5aMCatNLO#PDF_Choice_for_2017_production

Reply:
@perrozzi At the moment, mg26x branch does not have the gridpack_generation script and the MG5 patch (that allows many PDF sets) to add the recommended PDF sets fully. Should I just hard-code the PDF sets listed in the gridpack_generation in the master branch (need to drop some of them to fit to the limit of 25 sets)? Or is the branches going to be somehow merged soon?

Reply:
if I remember correctly, there is a python script to setup the variations in mg26x
@kdlong had started working also on that, not sure what the status is

Reply:
Yes, the idea is for this to be controllable from the python scripts. 

At the moment, you want the sample to be done with 2016 PDF sets? If so, I would leave the central PDF set as is and remove this line in runcmsgrid_NLO.sh before producing the gridpack:

https://github.com/cms-sw/genproductions/blob/mg26x/bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh#L100

For the 2017 sample the PDF list will be corrected shortly.

Reply:
We want the sample in the 2017 MC campaign, so we'll wait for the new PDF setup then. Thanks!